jpegenc/subblocks/bytestuffer/bytestuffer.py

```diff
@@ -0,0 +1,179 @@
+"""
+This module is MyHDL implementation
+of Byte Stuffer used for JPEG Encoder
+"""
+
+from myhdl import always_seq, always_comb
+from myhdl import Signal, concat, intbv, block
+
+
+class BSInputDataStream(object):
```

Comment:
@cfelton . Iam going to assign the `FIFOBus` output interface to the `Input Bus` shown. So, we dont need a FIFO Interface here, though we can add one.